doc/classes/EditorSettings.xml

```diff
@@ -412,6 +412,9 @@
 		<member name="editors/animation/onion_layers_past_color" type="Color" setter="" getter="">
 			The modulate color to use for "past" frames displayed in the animation editor's onion skinning feature.
 		</member>
+		<member name="editors/animation/default_bezier_key_behavior" type="int" setter="" getter="">
+			The default behavior for new bezier keys.
```

Comment:
```suggestion
			The default behavior for new bezier keys.
			- [b]Horizontal:[/b] Both tangents are fully horizontal regardless of the key's previous and next values.
			- [b]Balanced Automatic:[/b] Both tangents are rotated to match the key's previous and next values. The tangents' length is adjusted according to the distance between the previous and next values. If the key that is being added has no previous or next value, only one of the two tangents will be defined.
			- [b]Mirorred Automatic:[/b] Both tangents are rotated to match the key's previous and next values. The tangents' length is adjusted according to the distance between the previous and next values, but their length is forced to be equal.
```

Reply:
Done. Also added a description for the Linear option as that is added in now.